src/python/pants/backend/project_info/regex_lint.py

```diff
@@ -74,16 +74,25 @@ def from_dict(cls, d: dict[str, Any]) -> ValidationConfig:
 
 class RegexLintSubsystem(Subsystem):
     options_scope = "regex-lint"
-    help = (
-        "Lint your code using regex patterns, e.g. to check for copyright headers.\n\n"
-        "To activate this with the `lint` goal, you must set `[regex-lint].config`.\n\n"
-        "Unlike other linters, this can run on files not owned by targets, such as BUILD files. To "
-        "run on those, use `lint '**'` rather than `lint ::`, for example. Unfortunately, "
-        "`--changed-since=<sha>` does not yet cause this linter to run. We are exploring how to "
-        "improve both these gotchas."
+    help = softwrap(
+        """
+        Lint your code using regex patterns, e.g. to check for copyright headers.
+
+        To activate this with the `lint` goal, you must set `[regex-lint].config`.
+
+        Unlike other linters, this can run on files not owned by targets, such as BUILD files. To
+        run on those, use `lint '**'` rather than `lint ::`, for example. Unfortunately,
+        `--changed-since=<sha>` does not yet cause this linter to run. We are exploring how to
+        improve both these gotchas.
+        """
     )
-    schema_help = textwrap.dedent(
-        """\
+
+    _config = DictOption[Any](
+        "--config",
+        help=softwrap(
+            """
+            Config schema is as follows:
+
```

Comment:
You don't need the triple backticks on blocks indented 4 spaces. Markdown will render those as code blocks any way, and leaving them out makes gives less noise for the CLI help output ;)

Reply:
If we're going to make a stand on that (and maybe we should?) We should do it project-wide.

Reply:
I'm not sure which style we should go with, but agree that it makes sense to be consistent. I think the part I care about is indentation because it makes things more skimmable.

Reply:
Agreed. And I think less noise helps with skimmability, and if we can get rid of ` ``` ` then that's less noise too ;)